mlir/test/Dialect/Vector/invalid.mlir

```diff
@@ -1168,13 +1168,6 @@ func.func @reduce_unsupported_attr(%arg0: vector<16xf32>) -> i32 {
 
 // -----
 
-func.func @reduce_unsupported_third_argument(%arg0: vector<16xf32>, %arg1: f32) -> f32 {
-  // expected-error@+1 {{'vector.reduction' unsupported number of operands}}
```

Comment:
I'd just update the expected error to `expected ':'` rather remove the test

Reply:
What's the actual error that we will see here?